src/generate_e1.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+from __future__ import division
+import argparse
+import logging
+import numpy as np
+import os
+import pandas as pd
+import time
+
+from kaggler.data_io import load_data, save_data
+from kaggler.preprocessing import LabelEncoder
+
+# ref: https://www.kaggle.com/cuijamm/simple-onehot-logisticregression-score-0-80801
+def generate_feature(train_file, test_file, train_feature_file,
+                     test_feature_file, feature_map_file):
+    logging.info('loading raw data')
+    trn = pd.read_csv(train_file, index_col='id')
+    tst = pd.read_csv(test_file, index_col='id')
+
+    y_trn = trn['target']
+    del trn['target']
+    trn_tst = trn.append(tst)
+    n_trn = len(trn)
+    logging.info('trn_shape:{}, tst_shape: {}, all shape: {}'.format(trn.shape, tst.shape, trn_tst.shape))
+
+    logging.info('One Hot Encoding categorical variables')
+    trn_tst_ohe = pd.get_dummies(trn_tst, columns=trn_tst.columns, sparse=True)
```

Comment:
We can use `kaggler.preprocessing.OneHotEncoder` which supports grouping of rare values.